### PR TITLE
Create task02.py

### PR DESCRIPTION
**Pull request by MACE4GITHUB**
Learning objectives are done: 
- Basic operators and math module
- Control structures: if/else/elif, while loop, break
- Console input via input() and output via print()
- String parsing, f-strings

### CODE REVIEW

File: tasks/task02/task02.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+def show_error_message(message):
+    print('Invalid input. ' + message)
+
+def get_input_parts(message):
+    return input(message + ' ').split(' ')
+
+def get_base_and_height():
+    message = 'Enter base and height separated by a space:'
+    parts = get_input_parts(message)
+    while len(parts) != 2 or not all([parts[0].isdigit(), parts[1].isdigit()]):
```
Comment (hhalka) [inline tasks/task02/task02.py]:
```suggestion
    while len(parts) != 2 or not all(part.isdigit() for part in parts):
```

File: tasks/task02/task02.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        show_error_message('Please, enter the correct 2 sides and angle(degrees) between them separated by a space.') 
+        parts = get_input_parts(message)
+    return { 'side_one' : int(parts[0]), 'side_two' : int(parts[1]), 'angle' : int(parts[2]) }
+
+def get_area_by_two_sides_and_angle_betwenn_them():
+    params = get_two_sides_and_angle_between_them()
+    print_area((params['side_one'] * params['side_two'] / 2) * math.sin(math.radians(int(params['angle']))))
+
+def print_area(area):
+    print(f'Area is: {str(round(area))}')
```
Comment (hhalka) [inline tasks/task02/task02.py]: - you do not have to use `str` here
- it's not a good idea to use `round` as the result is not precise in this case

File: tasks/task02/task02.py (hunk 1 above)
Comment (MACE4GITHUB) [reply tasks/task02/task02.py]: Ok

File: tasks/task02/task02.py (hunk 2 above)
Comment (MACE4GITHUB) [reply tasks/task02/task02.py]: Ok

**Pull request merged**